frontend/ui/widget/container/framecontainer.lua

```diff
@@ -18,16 +18,18 @@ Example:
 
 --]]
 
-local WidgetContainer = require("ui/widget/container/widgetcontainer")
-local Geom = require("ui/geometry")
 local Blitbuffer = require("ffi/blitbuffer")
+local Device = require("device")
+local Geom = require("ui/geometry")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local Screen = Device.screen
```

Comment:
Nitpick, but if you only need screen then local Screen = require("device").screen is fine. ;-)